app/services/replication_transfer_updater.rb

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2015 The Regents of the University of Michigan.
+# All Rights Reserved.
+# Licensed according to the terms of the Revised BSD License
+# See LICENSE.md for details.
+
+class ReplicationTransferUpdater
+
+  def self.update(record, params)
+    run_update(build_update(record,params))
+  end
+
+  def self.build_update(record, params)
+    [CancelUpdate, FixityValueUpdate, StoreRequestedUpdate, StoredUpdate].each do |update_class|
+      return update_class.new(record, params) if update_class.matching_update?(record, params)
+    end
+    return ReplicationTransferUpdate.new(record, params)
+  end
+
+  def self.run_update(update)
+    update.update
+  end
```

Comment:
`run_update` is syperfluous as this one additional statement would be clearer if it were added to the end of the `update` method.  Unless `run_update` is actually called from anywhere else other than `update`?